### ISSUE TITLE
[HW2][Question] Use of UI components library

### ISSUE THREAD

**Issue by lucienmenassol** [swsnu/swppfall2017#36]
In the documentation of homework 2, you advise for using a css framework such as bootstrap, but is it possible to use a UI library of Angular components such as  [Material Angular](https://material.angular.io/)
or [PrimeNG](https://www.primefaces.org/primeng/#/) ?

(both are published under MIT license)

Edit: Also I was thinking about using npm+package.json to install one of them if it's possible to use one, so it would require for the person who do the grading to install them

**Comment by wonook** [comment]
Yes, both seems fine to me. I'll run `$ npm install` under each root directory before grading, so if the library installs with that command, it will be fine. Be sure to test it out with a clean copy of your project from your repository!

**Comment by lucienmenassol** [comment]
Ok, thank you for the answer !

**Comment by wonook** [comment]
Please don't close questions for the others! 😄

**Comment by lucienmenassol** [comment]
Oh sorry, I thought it would clutter the issue list.

For the labels I tried to, but unless I'm mistaken it seems I don't have the right to label issues on this repository (according to the github doc you have to have write access to do so).

**Comment by bgchun** [comment]
@lucienmenassol You can use the resolved label. I just created it.

**Comment by yuyupopo** [comment]
I used [ng-bootstrap](https://ng-bootstrap.github.io/#/home) UI components library for my project. 
Will it be okay? it's basically the same question above, but I wanted to make sure it's okay.

**Comment by wonook** [comment]
Yes, there won't be any restrictions regarding whatever css framework you use.
